packages/stateful/components/nft/NftSelectionModal.tsx

```diff
@@ -203,6 +209,30 @@ export const NftSelectionModal = ({
         nfts.errored ||
         nfts.data.length > 0 ? (
           <div className="mt-4 flex flex-col gap-4">
+            {unstakingDuration &&
+              ('height' in unstakingDuration
+                ? unstakingDuration.height
+                : unstakingDuration.time) > 0 && (
+                <div className="flex flex-row items-center gap-1">
```

Comment:
```suggestion
                <div className="-mt-3 flex flex-row items-center gap-1">
```

there's a bit too much spacing now, let's remove some